.gitignore

```diff
@@ -10,7 +10,8 @@ pids
 logs
 results
 
+node_modules
```

Comment:
no I'm saying YOU should put it in YOUR global gitignore [details how](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).  The only people this helps are people that haven't done that and just putting this here doesn't really help them long term because then they won't file an issue and learn all about global gitignores